rslib/src/notetype/schema11.rs

```diff
@@ -161,10 +161,7 @@ impl From<Notetype> for NotetypeSchema11 {
 
 /// See [crate::deckconfig::schema11::clear_other_duplicates()].
 fn clear_other_field_duplicates(other: &mut HashMap<String, Value>) {
-    for key in &["description"] {
-        other.remove(*key);
-    }
-    for key in &["plain_text"] {
+    for key in &["description", "plain_text"] {
```

Comment:
Like this?

Reply:
Pretty much, yep. For consistency with the other items like decks, recommend you change it to "plainText", and then update NoteFieldSchema11 like so:

#[derive(Serialize, Deserialize, Debug, Clone)]
#[serde(rename_all = "camelCase")]
pub struct NoteFieldSchema11 {